src/_includes/header-docs.html

```diff
@@ -0,0 +1,51 @@
+{%- include google-tag-manager-body.html -%}
+
+<header class="siteHeader2 siteHeader--docs">
+  <div class="siteHeader-logoAndMenuButton">
+    {% if page.layout == 'landing' %}
+    <!-- For the "Guide To App Devs" - /welcome_to_tilt.html -->
```

Comment:
I wonder if we should eliminate this page/layout at some point - it doesn't get a huge amount of views and people don't tend to stay on it particularly long.

(That said, it's got some inbound link sources, so we'd probably need a redirect or alternative)